roles/remote-user/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Determine whether to connect as root or admin_user
+  local_action: shell ssh -o PasswordAuthentication=no root@{{ groups.web.0 }} "echo root" || echo {{ admin_user }}
+  changed_when: false
+  register: remote_user
```

Comment:
The goal of this task is to determine the appropriate `remote_user` for `server.yml`'s main play. That play will run on `hosts: web`, so this task tests whether `root` can connect to the first host in the `web` group. The hosts in this group are accessible in the ansible variable `groups.web`. 

I will replace `root@{{ groups.web.0 }}` with `root@{{ groups.web | first }}` to switch from a hard-coded feel to something that may communicate the concept more intuitively. Suggestions welcome.

Aside: There's a rare potential that the `web` group could consist of multiple hosts, added at different times, some permitting `root` login while others forbid it. I commented on this scenario in #246 ("Conditional remote_user" section).